src/sphinx_tags/__init__.py

```diff
@@ -178,6 +178,7 @@ def create_file(
             content.append("#" * textwidth(header))
             content.append("")
             #  Return link block at the start of the page"""
+            content.append(f".. _sphx_tag_{self.file_basename}_{self.name}:")
```

Comment:
I believe we only need 
```suggestion
            content.append(f".. _sphx_tag_{self.file_basename}:")
```

since `file_basename` will be normalized (with no emojis or special characters) and `name` will contain all those. 

I have a github.com/melissawm/minimalsphinx project and used it to test this. Your suggestion may be enough and honestly I'm not sure if there's a better option.

Thanks again!

Reply:
Thank you, changes made and I added a bit in the docs.